### ISSUE TITLE
getting logits from a model

### ISSUE THREAD

**Issue by Sidd1609** [ultralytics/yolov5#11707]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

How to get logits for an inference image before the classification of classes happen, how to write an inference script where I could see the outputs at each layer?

**Comment by glenn-jocher** [comment]
@Sidd1609 to get the model's logits for an inference image before classification, you can access the raw model output by using the `model.forward()` method. To see the outputs at each layer, you may want to look at the model's `model.yaml` file for the model architecture and hook into the layers you are interested in. Feel free to check the YOLOv5 documentation for more information on custom inference scripts and model outputs. Let me know if you need further assistance!
